tools/buildgen/plugins/generate_vsprojects.py

```diff
@@ -25,9 +25,8 @@
 def mako_plugin(dictionary):
     """The exported plugin code for generate_vsprojeccts
 
-  We want to help the work of the visual studio generators.
-
-  """
+    We want to help the work of the visual studio generators.
```

Comment:
Since you're touching the buildgen logic, I think  the "generate_vsprojects.py" plugin is no longer needed and can be deleted.